bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import random
+from datetime import datetime
+from typing import Optional
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+from bot.constants import NEGATIVE_REPLIES
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches info from GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @staticmethod
+    async def fetch_data(url: str) -> dict:
+        """Retrieve data as a dictionary."""
+        async with aiohttp.ClientSession() as cs:
+            async with cs.get(url) as r:
+                return await r.json()
+
+    @commands.command(name='github', aliases=['gh'])
+    @commands.cooldown(1, 5, BucketType.user)
+    async def get_github_info(self, ctx: commands.Context, username: Optional[str]) -> None:
+        """
+        Fetches a user's GitHub information.
+
+        Username is optional and sends the help command if not specified.
+
+        Usage: .gh [username]
+        """
+        if username is None:
+            await ctx.invoke(self.bot.get_command('help'), 'github')
+            return
+
+        async with ctx.typing():
+            user_data = await self.fetch_data(f"https://api.github.com/users/{username}")
+
+            # User_data will not have a message key if the user exists
+            if user_data.get('message') is not None:
+                await ctx.send(embed=discord.Embed(title=random.choice(NEGATIVE_REPLIES),
+                                                   colour=discord.Colour.red()))
+                return
+
+            org_data = await self.fetch_data(user_data['organizations_url'])
+            orgs = [f"[{org['login']}](https://github.com/{org['login']})" for org in org_data]
+            orgs_to_add = ' | '.join(orgs)
+
+            starred_data = await self.fetch_data(user_data['starred_url'])
+
+            # Forming blog link
+            if user_data['blog'].startswith("http"):  # Blog link is complete
+                blog = f"[Direct link]({user_data['blog']})"
+            elif user_data['blog']:  # Blog exists but the link is not complete
+                blog = f"[Direct link](https://{user_data['blog']})"
```

Comment:
Instead of hyperlinking `Direct link`, could we just have the link itself? I think that would look best.
```suggestion
                blog = user_data['blog']
            elif user_data['blog']:  # Blog exists but the link is not complete
                blog = f"https://{user_data['blog']}"
```

It'd look something like this:
![image](https://user-images.githubusercontent.com/52584399/95517182-8b2e3780-098e-11eb-94d5-5fb3b0b155a5.png)